test/e2e/e2e_test.go

```diff
@@ -59,6 +59,7 @@ var _ = Describe("E2e", Ordered, func() {
 			Method:  "DELETE",
 			Verbose: true,
 			App:     "curl",
+			Headers: []string{"token: test"},
```

Comment:
Do you mind making expected token value a const so updates to either passed in string or expected won't break things?